components/ListItem/ListItem.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {ListContainer} from './ListItem.Container';
+import {styles} from './ListItem.Styles';
+import {FlatList, Text, View} from 'react-native';
+
+const ListItem = () => {
+  return <View style={styles.container}></View>;
+};
+
+export default ListItem;
```

Comment:
Do not export by default the component, if you do this you give the chance to renamed the component when is imported.